pandas/core/computation/align.py

```diff
@@ -10,7 +10,9 @@
 import pandas as pd
 from pandas import compat
 from pandas.errors import PerformanceWarning
-from pandas.core.common import flatten
+
+import pandas.core.common as com
```

Comment:
I really don't like extra blank lines between imports for the most part, occasionally for readibility